extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -216,9 +217,10 @@ function SidebarWithRef(
           size={false}
           layout={true}
           paint={true}
-          className={`${classes.sidebarClass} ${
-            side === 'left' ? classes.left : classes.right
-          }`}
+          part="sidebar"
+          wrapperClassName={`${classes.sidebarClass} ${
+            classes.defaultSidebarStyles
```

Comment:
`wrapperClassName` and `className` seem to do the same thing in the Contain component?